### ISSUE TITLE
[Feature Request] Multilingual spellchecker

### ISSUE THREAD

**Issue by RickStanley** [telegramdesktop/tdesktop#10103]
### Describe the solution you'd like
I would like to have the option to have spellchecking for multiple languages, and if possible simultaneous spellchecking.

### Describe alternatives you've considered
No alternatives. Currently, you would have to change the current language of Telegram's interface.

### Additional context
From a developer's perspective, international communication with clients, corporates and/or group chats is very common (at least for me). From time to time, I may forget some words that might be wrong but I remember the basic syllables structure and wait for the spellchecker to suggest the closest option.

We already have a spellcheck option at disposal:
![spellcheck option from telegram](https://user-images.githubusercontent.com/23178554/104143904-b5313480-5397-11eb-913d-4d3d35ef2d33.png)

Would be nice to have the option to opt-in for multilingual (and maybe simultaneous) checking there. Like this (from Microsoft Edge):
![microsoft edge's multilingual spellcheck support options](https://user-images.githubusercontent.com/23178554/104144045-430d1f80-5398-11eb-9349-50712fde723c.png)

Basically a set of available spellchecking for languages chosen by the user.

----
English is not my mother tongue, I hope I didn't make any mistakes it would be ironic. xD

**Comment by ilya-fedin** [comment]
Telegram uses your system spellchecker, you should setup it in your system settings.
